src/Plugin/PhpUnit.php

```diff
@@ -58,11 +58,14 @@ public static function pluginName()
 
     /**
      * Standard Constructor
-     * $options['config']    Path to a PHPUnit XML configuration file.
-     * $options['run_from']  The directory where the phpunit command will run from when using 'config'.
-     * $options['coverage']  Value for the --coverage-html command line flag.
-     * $options['directory'] Optional directory or list of directories to run PHPUnit on.
-     * $options['args']      Command line args (in string format) to pass to PHP Unit
+     * $options['config']                       Path to a PHPUnit XML configuration file.
+     * $options['run_from']                     The directory where the phpunit command will run from when using 'config'.
+     * $options['coverage']                     Value for the --coverage-html command line flag.
```

Comment:
Could you add new parameters to plugin documentation (https://github.com/php-censor/php-censor/blob/master/docs/en/plugins/php_unit.md) please?

Reply:
Not a problem.